py/runai_model_streamer/runai_model_streamer/file_streamer/requests_iterator.py

```
@@ -23,11 +39,19 @@ class MemoryCapMode(enum.Enum):
     largest_chunk = 3
 
 class FileChunks:
-    def __init__(self, id: int, path: str, offset: int, chunks: List[int]) -> None:
+    def __init__(self, id: int, path: str, offset: int, chunks: List[int],
```

Ruff (0.15.6)

[error] Function argument `id` is shadowing a Python builtin

(A002)